src/hooks/usePreviousModalRoute.ts

```diff
@@ -0,0 +1,23 @@
+/*
+Hook used to preserve the underlying component route while navigating modal routes
+*/
+
+import { useEffect, useRef } from 'react';
+
+interface LocationType {
+  pathname: string;
+  search?: string;
+}
+
+const usePreviousModalRoute = (location: LocationType, route: string) => {
```

Comment:
Can you add a comment explaining what these params are intended to be?

Reply:
https://github.com/CMSgov/mint-app/pull/480/commits/a39acdaf6c6e27bd82b86b2e942f1066e35cc49a